### DISCUSSION TITLE
JDownloader2

### DISCUSSION THREAD

**Discussion by uasaad** [catppuccin/catppuccin#2488] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Hey :wave:,

I've closed this discussion since the original post doesn't include any links on how to theme the application or link to any source code of other themes being applied to the application.

Would be more than happy to re-open once the post is updated with the relevant links or instructions on how to theme it. 

Thanks!

**Reply by TheGhostHybrid** [reply]
Hi there! The colors can be accessed via the Advanced Settings within the general Settings. Doing a search for “laf color” pulls up all the elements that can be customized. It’d be a bit tedious, but totally doable!

In addition to that, it’s a Java application. If there is any way to autotheme Java applets, it might work here.

<img width="3840" height="2160" alt="image" src="https://github.com/user-attachments/assets/f3b5b398-465d-4c1c-8d83-969acc8926d9" />
